File: backend/src/main/java/com/scrs/pojo/Student.java
package com.scrs.pojo;/*
 * @date 12/04 21:22
 */

import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

@Data
@TableName("student")
public class Student {
    private Integer id;

    private String sname;

    private String password;

    private String sex;

    private Integer age;

    private String major;

    private String college;

    private String simage;
}

File: frontend/src/store/student.ts
import { defineStore } from 'pinia';

export const useStudentStore = defineStore('student', {
  state: () => ({
    student: {
      id: null,
      sname: '',
      sex: '',
      age: null,
      major: '',
      college: '',
      simage: '',
      password: '',
    },
  }),
  actions: {
    setStudent(student:any) {
      this.student = student;
    },
  },
});